source/ed/connectors/fusio_parser.h

```diff
@@ -44,6 +44,8 @@ www.navitia.io
 namespace ed {
 namespace connectors {
 
+static const int unknown_column = -1;
```

Comment:
For consistency, do you plan (not this PR I guess) on extending the use of it? As the same check is done on other places.
We try to use UPPER_CASE for static var:
```suggestion
static const int UNKNOWN_COLUMN = -1;
```

Reply:
Both are good for me :smiley: 
Both exists in the current code